extensions/parallels-virtual-machines/src/utils.ts

```diff
@@ -44,12 +46,48 @@ function stateToDescription(state: VMState): string {
 
 function iconForVM(vm: VM): Image {
   let icon = "other";
+  let colortype = "mono";
+  if (parseInt(getPreferenceValues().IconColorType) === IconsColorType.Color) {
+    colortype = "colored";
+  }
```

Comment:
I think if we don't use a number-as-string, we can use the names directly and skip parsing and this conditional entirely.